src/controller/java/AndroidCallbacks.cpp

```diff
@@ -165,10 +166,6 @@ ReportCallback::~ReportCallback()
         env->DeleteGlobalRef(mSubscriptionEstablishedCallbackRef);
     }
     env->DeleteGlobalRef(mReportCallbackRef);
-    if (mReadClient != nullptr)
-    {
-        Platform::Delete(mReadClient);
-    }
```

Comment:
You should probably just leave this block as it was.

Reply:
yes, agree, updated, thanks